README.md

```diff
@@ -72,11 +72,12 @@ If you compile RIOT for the native cpu and include the `netdev_tap` module,
 you can specify a network interface like this: `PORT=tap0 make term`
 
 #### SETTING UP A TAP NETWORK
-There is a shellscript in `RIOT/dist/tools/tapsetup` called `tapsetup` which
+There is a shell script in `RIOT/dist/tools/tapsetup` called `tapsetup` which
 you can use to create a network of tap interfaces.
 
-*USAGE*
-To create a bridge and two (or count at your option) tap interfaces:
+_USAGE_
+
+To create a bridge and two (or _count_ at your option) tap interfaces:
 
     ./dist/tools/tapsetup/tapsetup [-c [<count>]]
```

Comment:
as you are at it remove superfluous brackets around `<count>`

```
./dist/tools/tapsetup/tapsetup [-c <count>]
```

Reply:
These are not superfluous. You can either write

```sh
./dist/tools/tapsetup/tapsetup      # creates two TAPs, since it defaults to `-c`
./dist/tools/tapsetup/tapsetup -c   # also creates two TAPs, since it defaults to `-c 2`
./dist/tools/tapsetup/tapsetup -c 4 # creates four TAPs
```

Reply:
mhm, I see - a bit weird though ...

then leave as is

Reply:
What is weird about it? It's normal documentation of command-line parameters (have a look at any manpage of a command of optional parameters with optional arguments).

Reply:
oh I didn't mean the documentation, but rather the option handling of the script - which only means I (wrongly) assumed it works differently, never mind ...